src/components/about.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const About = () => {
+  return (
+    <>
```

Comment:
I understand that this "tag" works good but please try to use a div or React.Fragment instead.

```suggestion
    <div>
```